spin/apps/02b-json-api-openai/spin.toml

```
@@ -0,0 +1,23 @@
+spin_manifest_version = "1"
+authors = ["teknicus"]
+description = ""
+name = "magic-8-ball"
+trigger = { type = "http", base = "/" }
+version = "0.1.0"
+
```

Like you mentioned, this would be even better using config to enable users to set their key using a provider.
```suggestion
[variables]
openai_key = { required = true }
```